clickhouse/columns/string.cpp

```diff
@@ -179,18 +137,21 @@ ColumnString::ColumnString(const std::vector<std::string> & data)
     }
 }
 
-ColumnString::~ColumnString()
-{}
+ColumnString::ColumnString(std::vector<std::string>&& data)
```

Comment:
Please add a test for this constructor.

Reply:
hi, ColumnsCase.StringInit has covered yet.